force_wfmanager/left_side_pane/data_source_model_view.py

```diff
@@ -82,6 +84,7 @@ def update_name(self):
 output_slots_editor = TableEditor(
     sortable=False,
     configurable=False,
+    selected="selected_slot_row",
```

Comment:
it felt a bit weird to have the same selected entry for two tables, but apparently table editor is smart enough to unselect the other one when it's not in the list.